lnst/Tests/LongLivedConnections.py

```diff
@@ -0,0 +1,174 @@
+from .BaseTestModule import BaseTestModule
+import socket
+import logging
+import threading
+import time
+import select
+import resource
+from typing import Union
+
+from lnst.Common.IpAddress import Ip4Address, Ip6Address, BaseIpAddress
+from lnst.Common.Parameters import IpParam, IntParam, ListParam, StrParam
+
+
+def get_inet(ip: Union[Ip4Address, Ip6Address]):
+    return socket.AF_INET
+    if isinstance(ip, Ip4Address):
+        return socket.AF_INET
+    elif isinstance(ip, Ip6Address):
+        return socket.AF_INET6
+    else:
+        raise ValueError(f"Unknown IP address type: {ip}")
+
+
+class LongLivedMixin(BaseTestModule):
```

Comment:
Why we call this a mixin ? To me this seems to be just a base class for the server, client implementations.

I'd understand if we do inheritance with:
```
class LongLivedMixin():
   pass

class LongLivedServer(BaseTestModule, LongLivedMixin):
   pass

class LongLivedClient(BaseTestModule, LongLivedMixin):
   pass
```